src/Upload.js

```diff
@@ -95,6 +95,7 @@ export default class Upload {
       debug(` - End: ${end}`)
 
       const res = await safePut(opts.url, chunk, { headers })
+      this.last_res = res;
```

Comment:
Casing doesn't match the rest of this project - can you use camel case for this please?

Reply:
Yes, sure thing.

I also figured it's a good idea to initialise the variable to null, rather than undefined.